Randolph-dev/Druid_Partnering_Project_React24k
# SCRUM-63 Feat:add frontpage content

### Related ticket:
[SCRUM-63](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-63)

### In this PR:

- Added frontpage content in drupal
- It has two fields. Description and link
- All the text is in the description but I added the images as a link

### Testing Instructions
git pull origin
lando db-import SCRUM-63
Login to your drupal site and check whether the changes have been appliend

[SCRUM-63]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-63?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
